deeppavlov/models/bert/bert_sequence_tagger.py

```diff
@@ -13,51 +13,177 @@
 # limitations under the License.
 
 from logging import getLogger
-from typing import List, Union, Dict
+from typing import List, Any, Tuple, Union, Dict, Optional
 
 import numpy as np
 import tensorflow as tf
+from tensorflow.python.ops import array_ops
 from bert_dp.modeling import BertConfig, BertModel
 from bert_dp.optimization import AdamWeightDecayOptimizer
 
 from deeppavlov.core.commands.utils import expand_path
 from deeppavlov.core.common.registry import register
+from deeppavlov.core.data.utils import zero_pad
+from deeppavlov.core.models.component import Component
 from deeppavlov.core.layers.tf_layers import bi_rnn
 from deeppavlov.core.models.tf_model import LRScheduledTFModel
 
 log = getLogger(__name__)
 
 
-@register('bert_ner')
-class BertNerModel(LRScheduledTFModel):
-    """BERT-based model for text tagging.
+def token_from_subtoken(units: tf.Tensor, mask: tf.Tensor) -> tf.Tensor:
+    """ Assemble token level units from subtoken level units
 
-    For each token a tag is predicted. Can be used for any tagging.
+    Args:
+        units: tf.Tensor of shape [batch_size, SUBTOKEN_seq_length, n_features]
+        mask: mask of token beginnings. For example: for tokens
+
+                [[`[CLS]` `My`, `capybara`, `[SEP]`],
```

Comment:
```suggestion
                [[``[CLS]`` ``My``, ``capybara``, ``[SEP]``],
```
everywhere. Or maybe wrap these blocs in `.. code:: python`